pkg/controllers/subnetset/namespace_handler.go

```diff
@@ -0,0 +1,43 @@
+package subnetset
+
+import (
+	"context"
+
+	"k8s.io/client-go/util/retry"
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+
+	"github.com/vmware-tanzu/nsx-operator/pkg/apis/v1alpha1"
+)
+
```

Comment:
It's better to add some comments to explain why we should add extra handler, like [https://github.com/vmware-tanzu/nsx-operator/blob/main/pkg/controllers/securitypolicy/pod_controller.go#L19](url)

Reply:
Besides, the name handler is suitable, but how about complying with the security policy and renaming it to controller?

Reply:
NamespaceHandler is removed and comments are added to VPCHandler and SubnetPortHandler. As for the name, I prefer `handler` personally as `handler` is considered as a part of `controller/reconciler`.